openedx/adg/lms/webinars/admin.py

```diff
@@ -29,6 +30,12 @@ def save_model(self, request, obj, form, change):
             obj.modified_by = request.user
         obj.save()
 
+    def change_view(self, request, object_id, form_url='', extra_context=None):
+        if '_saveasnew' in request.POST:
+            request.POST = request.POST.copy()
+            request.POST['status'] = Webinar.UPCOMING
```

Comment:
By default POST is immutable, we are making it mutable and not for just this method but for its life time. Here are the two alternatives
- Move it on front in javascript, we have already overrode the template we can add our js in it, directly listen, click on, save as button and remove the status. Then on backend default status is UPCOMING. So we won't need to override this method.
- Change the mutability at line 35 and revert it back once we have changes the status on line 36. But still first approach is much cleaner and better.

Reply:
Good catch (Y)

1.  I tried the first option made the change in javascript , but there is a issue djangoadmin is using the same clone button before cloning and on submission of cloned object because of this our clear status method will call 2 times and user will not able to submit the form.

2. The option to is fine i made the change and thats working fine. 
`request.POST._mutable = False`